### PR TITLE
feat: Safe Wallet improvements

### PR DESCRIPTION
**Pull request by chybisov**

### CODE REVIEW

File: src/utils/iframe.ts
Comment (dennyscode) [inline src/utils/iframe.ts:1]: That´s interesting!

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+  },
+  "[javascript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "[json]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "[jsonc]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  }
```
Comment (dennyscode) [inline .vscode/settings.json:20]: Nice! That way, we should be more in sync in terms of formatting, right? Currently, I believe we all have different settings where just saving a file on one machine affects the existing code differently than on another ones machine 😄

File: src/i18n/translations/en/translation.json [hunk 2]
```diff
@@ -112,10 +112,6 @@
     "transactionInitiated": {
       "title": "Multiple signatures required",
       "description": "Please notify other multisig wallet participants to sign before the transaction expires."
-    },
-    "alert": {
-      "title": "Multisig wallet connected",
-      "description": "Destination wallet address is required to complete the exchange."
```
Comment (dennyscode) [inline src/i18n/translations/en/translation.json:118]: Nice, you removed that as well!

**Review by dennyscode — APPROVED** [review]

**Review by oktapodia — APPROVED** [review]
LGTM

**Review by tcheee — APPROVED** [review]

**Pull request merged**